examples/personality_insights_v3.py

```diff
@@ -17,7 +18,7 @@
 
 # Authentication via external config like VCAP_SERVICES
 service = PersonalityInsightsV3(version='2017-10-13')
-service.set_service_url('https://gateway.watsonplatform.net/personality-insights/api')
+service.set_service_url('https://api.us-east.personality-insights.watson.cloud.ibm.com/instances/4c18b521-3abd-4c7c-bec7-6a3fd03644f1')
```

Comment:
I don't think you need to show the instance url here. You can use the regular us-east url
```
https://gateway-wdc.watsonplatform.net/personality-insights/api
```

Reply:
There is a bug in recent python core: https://github.com/IBM/python-sdk-core/issues/54. Matthew Keezer said he will work on it